libraries/registration-api-client/pom.xml

```diff
@@ -51,6 +51,11 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-rest</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Noen grunn til at du bruker 2.8.0 i stedet for 2.8.5 som er latest ?  https://mvnrepository.com/artifact/com.google.code.gson/gson